Lmod/branches/7.0/Lmod-illumina.spec

```diff
@@ -5,7 +5,7 @@
 %define release         cm7.0
 %define name            Lmod
 %define secname         Lmod-files
-%define version         6.6
+%define version         7.1
```

Comment:
@johnnydevaprasad : could you provide a refreshed set of RPMs at 7.3.16 or so? (cm7{0,3})
I'm trying to isolate a bug and it would help me to run one more round of checks!